lib/external: Add parson library for JSON (#3028)
OSGeo/grass: lib/external/README.md

# How to add a new external library

> The follow procedure will walk you through adding a new third-party

Reviewer: ```suggestion
The following procedure will walk you through adding a new third-party
```

library to GRASS GIS.

## License

> Before you begin make sure the library you want to include is complient with
> GRASS GIS's license.

Reviewer 2: ```suggestion
Before you start, make sure that the library you want to include is in
compliance with the GRASS GIS licence.
```

### Approved Third-Party Licenses

* MIT/X
* GPLv2
* LGPL 2.1
* MPL 2.0
* BSD-3-Clause
* Public Domain

## Procedure

* Create a directory that will contain the new library's files
in `lib/external/<new-library>`

> * Add the new libraries header, c, license, and README to the newly

Reviewer 2: "c" is unclear to me -> "copyright"?

Reviewer: ```suggestion
* Add the new libraries header and source files, license, and README to the newly
```
